### DISCUSSION TITLE
How can i check available updates with commits?

### DISCUSSION THREAD

**Discussion by ogios** [mason-org/mason.nvim#1532] [Q&A] [answered]
currently it only shows available updates with tag, but some commits contains the feat or fix that i need urgently and they haven't create a new tag with release.
so i want to know if there's a way to check available updates with commits to get immediate update.

**Comment by williamboman** [comment, marked as answer]
Mason tracks upstream versions and not individual commits. You can however specify a commit as the version when installing a package, for example: `:MasonInstall package@7d48559`.

**Reply by ogios** [reply]
thanks
